### ISSUE TITLE
Update rocker/rstudio image on dockerhub

### ISSUE THREAD

**Issue by vpanfilov** [rocker-org/rocker-versioned2#101]
Hello.

There was a major release of RStudio (v1.4.1103) on January 14th but images on Dockerhub were built about a month ago with RStudio Sever v1.3.1093. Can you push updated rocker/rstudio image?

**Comment by Mishco** [comment]
I agree, it will be very nice to have image with R>4.0.* and latest RStudio https://rstudio.com/products/rstudio/release-notes/

**Comment by nathanweeks** [comment]
I noticed rocker/geospatial:4.0.4 is currently RStudio 1.4.1106, but [rocker/binder:4.0.4](https://hub.docker.com/layers/rocker/binder/4.0.4/images/sha256-4bc2edd7f936d7c1f109b51eda66e6a80714558f2b20d6408044e47150a1970a), which supposedly uses rocker/geospatial:4.0.4 as its base image, is RStudio 1.3.959?

**Comment by eitsupi** [comment]
This issue has been out of date for over a month.

**Comment by eitsupi** [comment]
Now that #164 has been merged, when a new RStudio is released, the bot will respond within a day and create a PR with an updated Dockerfile, and build the images as soon as the PR is merged.
This should prevent this RStudio update delay problem from occurring in the future.
